lib/pipe/html.js

```diff
@@ -222,17 +222,45 @@ class HtmlPipe {
         </select>`,
         ),
     );
-
+    /* eslint-disable */
     handlebars.registerHelper(
       'emptyDataComponent',
       () =>
         new handlebars.SafeString(
-          `<div class="noData m-0">
+          `
+          <div class="noData">
+          <div class="noDataSvg">
+          <svg width="185" height="185" viewBox="0 0 370 370" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <g clip-path="url(#clip0_215_375)">
+          <path opacity="0.1" d="M185 369.998C287.173 369.998 370.001 287.171 370.001 184.999C370.001 82.8268 287.173 0 185 0C82.8274 0 0 82.8268 0 184.999C0 287.171 82.8274 369.998 185 369.998Z" fill="#A9C7FF"/>
```

Comment:
I think we need another file with svg extension here
probably we can add that to `template` dir